mis_builder/views/mis_report_instance.xml

```diff
@@ -81,18 +81,22 @@
                     <notebook>
                         <page string="Columns" attrs="{'invisible': [('comparison_mode', '=', False)]}">
                             <group>
-                                <field name="period_ids" colspan="4" nolabel="1" attrs="{'required': [('comparison_mode', '=', True)]}" context="{'default_report_instance_id': id}">
-                                    <tree colors="red:valid==False">
-                                        <field name="sequence" widget="handle"/>
-                                        <field name="name"/>
-                                        <field name="source"/>
-                                        <field name="date_from"/>
-                                        <field name="date_to"/>
-                                        <field name="valid" invisible="1"/>
-                                    </tree>
-                                </field>
-                                <field name="date"/>
+                                <group>
+                                    <field name="date" />
+                                </group>
+                                <group>
+                                </group>
                             </group>
+                            <field name="period_ids" colspan="4" attrs="{'required': [('comparison_mode', '=', True)]}" context="{'default_report_instance_id': id}">
+                                <tree colors="red:valid==False">
+                                    <field name="sequence" widget="handle"/>
+                                    <field name="name"/>
+                                    <field name="source"/>
+                                    <field name="date_from"/>
+                                    <field name="date_to"/>
+                                    <field name="valid" invisible="1"/>
+                                </tree>
+                            </field>
```

Comment:
Move it inside the above group to remove the empty space between the date and the list?

Reply:
I'll try

Reply:
@sbidoul Done